#!/usr/bin/env python
PACKAGE = "cora_helm"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed_limiter",        double_t, 0, "Limit speed for tight quaters.",                  0.15,    0.0,   1.0)
gen.add("turn_speed_limiter",   double_t, 0, "Limit speed used for differential turning.",      0.3,     0.0,   1.0)
gen.add("differential_bias",    double_t, 0, "How much to favor fwd thrust while turning.",     0.44,    0.0,   1.0)
gen.add("magnetic_declination", double_t, 0, "Difference between magnetic and true north.",     0.0,  -180.0, 180.0)

exit(gen.generate(PACKAGE, "cora_helm", "cora_helm"))

